games-arcade/vvvvvv/vvvvvv-2.0~git.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Change gravity (instead of jumping) to save your crew"
+DESCRIPTION="Notice requested by the author: VVVVVV is a commercial game! The author has \
+given special permission to make this Haiku version available for free. If you \
+enjoy the game, please consider purchasing a copy at thelettervsixtim.es. \
+
+VVVVVV is a hit 2010 indie video game from Super Hexagon creator Terry Cavanagh. \
+Play as Captain Viridian, whose must rescue their crew after a teleporter mishap \
+caused them to be seperated in Dimension VVVVVV."
```

Comment:
Maybe:
```
SUMMARY="A 2D puzzle-platform game"
DESCRIPTION="Notice requested by the author: \
VVVVVV is a commercial game! The author has given special permission \
to make this Haiku version available for free. If you enjoy the game, \
please consider purchasing a copy at thelettervsixtim.es.

VVVVVV is a hit 2010 indie video game from Super Hexagon creator Terry \
Cavanagh.
Play as Captain Viridian, whose must rescue their crew after a teleporter \
mishap caused them to be seperated in Dimension VVVVVV."
```

Reply:
Yeah the original description was similar to that, but I changed it based on @extrowerk's feedback:
https://github.com/haikuports/haikuports/pull/4652#discussion_r371029661

Reply:
Original description was similar to that, but I changed it based on feedback from @extrowerk.

Reply:
Current one still exceeds the 80 char limit in a few lines, also (as i stated before), no backslashes when you want to start a new line (looks better in HaikuDepot)

Reply:
How about this instead:
> A jumpless, gravity-flipping, 2D puzzle-platformer

Still describes what kind of game it is, while briefly telling what differentiates it from others in the same genre.

Reply:
https://aur.archlinux.org/packages/vvvvvv (same summary shows up at gentoo also)

Reply:
PS, updated a bit

Reply:
![vvvvvv-2](https://user-images.githubusercontent.com/16057090/73183370-fb579e80-411a-11ea-9b93-210f4cdd15c8.png)